app/models/repository_row.rb

```diff
@@ -0,0 +1,14 @@
+class RepositoryRow < ActiveRecord::Base
+  belongs_to :repository
+  belongs_to :team
```

Comment:
Do we need `belongs_to :team`? I mean, you can get this from the repository? Is there any optimization benefit from this?